testing/test_pytest_html.py

```diff
@@ -305,7 +305,12 @@ def pytest_runtest_makereport(item, call):
             content)
         assert link in html
 
-    def test_extra_image(self, testdir):
+    @pytest.mark.parametrize('mime_type, extension',
+                             [('image/png', 'png'),
+                              ('image/png', 'image'),
+                              ('image/jpeg', 'jpg'),
+                              ('image/svg+xml', 'svg')])
+    def test_extra_image(self, testdir, mime_type, extension):
```

Comment:
Thanks for making this parameterised. I wonder though if we could actually have a new test `test_extra` to replace `test_extra_json`, `test_extra_image`, and `test_extra_text`? It would be similar to `test_extra_separated`, which itself could be enhanced to include the new image types. What do you think?